magpie/ui/utils.py

```diff
@@ -21,8 +22,8 @@ def request_api(request,            # type: Request
                 cookies=None,       # type: Optional[CookiesType]
                 ):                  # type: (...) -> Response
     """
-    Use a pyramid sub-request to request Magpie API routes via the UI.
-    This avoids max retries and closed connections when using 1 worker (eg: during tests).
+    Use a pyramid sub-request to request Magpie API routes via the UI. This avoids max retries and closed connections
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'str'](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650807)